spec/src/main/asciidoc/core/implementation.asciidoc

```diff
@@ -18,7 +18,7 @@ Portable extensions and build compatible extensions may provide other kinds of b
 
 A _managed bean_ is a bean that is implemented by a Java class.
 This class is called the _bean class_ of the managed bean.
-The basic lifecycle and semantics of managed beans are defined by the Managed Beans specification.
+The basic lifecycle and semantics of managed beans are defined by the CDI specification.
```

Comment:
I would have to look at the Managed Beans spec, but is there maybe something we should copy to CDI?

... Let me check ...

Reply:
Well it seems to me that the Managed Beans specification doesn't really specify anything interesting, and if memory serves well, CDI redefines everything anyway. I think this whole sentence

> The basic lifecycle and semantics of managed beans are defined by the Managed Beans specification.

could be just dropped.

Reply:
Removed